module/Application/src/Service/WatermarkService.php

```diff
@@ -51,7 +51,7 @@ public function watermarkPdf(string $path): string
         $drawSettings->setFont(self::FONT);
         $drawSettings->setTextAlignment(Imagick::ALIGN_CENTER);
         $drawSettings->setFillColor($fillPixelLight);
-        $drawSettings->setFillOpacity(0.20);
+        $drawSettings->setFillOpacity(0.40);
```

Comment:
Doesn't this affect the readability too much?

Reply:
I personally do not have issues with it being `0.40`, but we should probably go to `0.30` to ensure it stays readable for everyone (`0.20` makes it unintelligible when the background is not white).